worker/pkg/workflows/datasync/activities/activities_test.go

```diff
@@ -502,10 +509,18 @@ func Test_buildProcessorConfigJavascriptMultiple(t *testing.T) {
 		{Schema: "public", Table: "users", Column: col2, Transformer: &mgmtv1alpha1.JobMappingTransformer{Source: jsT2.Source, Config: jsT2.Config}}})
 
 	assert.NoError(t, err)
-	constructedCode := constructJavascriptCode(code, col)
-	constructedCode2 := constructJavascriptCode(code2, col2)
-	joinedCode := fmt.Sprintf("%s\n%s", constructedCode, constructedCode2)
-	assert.Equal(t, fmt.Sprintf(`(() => {%s})();`, joinedCode), res.Javascript.Code)
+
+	jsFunctions := []string{}
+	benthosOutputs := []string{}
+
+	benthosOutput := constructBenthosOutput(col)
+	benthosOutput2 := constructBenthosOutput(col2)
+	benthosOutputs = append(benthosOutputs, benthosOutput, benthosOutput2)
+	jsFunction := constructJsFunction(code, col)
+	jsFunction2 := constructJsFunction(code2, col2)
+	jsFunctions = append(jsFunctions, jsFunction, jsFunction2)
+	constructedProcessor := constructBenthosJsProcessor(jsFunctions, benthosOutputs)
+	assert.Equal(t, constructedProcessor, res.Javascript.Code)
```

Comment:
you thought it would be this easy, huh? :)

So some things about this need to change.

This test will basically never fail because the `constructedProcessor` variable here is using the same methods that `buildProcessorConfig` uses to construct `res.Javascript.Code`. So they will guaranteed to always be equivalent (because they all use the same methods!).
Now, this method would fail of course if a third column was dded and the `jsFunctions` here didn't include that. But this should be testing the output with a raw string of what you (the tester) are expecting the output to be without any of the helper functions. 

That's why in many of our other tests we run the builder function and assert it against a hardcoded benthos config string.

So these lines should really be replaced with something like:

```
assert.Equal(
  t,
  `your-expected-constructed-processor-string`,
  res.Javascript.Code,
)

Reply:
i was hoping it would be :) updated it